package.json

```diff
@@ -39,7 +39,8 @@
     "test:playwright:a11y:local": "USE_PROXY=false PW_WIDTH_VALUE=768 PW_HEIGHT_VALUE=720 PW_BROWSER_VALUE=chromium yarn playwright test --project=a11y",
     "test:types": "yarn build:env-loader && APP_ENV=test tsc --noEmit",
     "storybook": "storybook dev -p 6006",
-    "storybook:build": "storybook build -o out/storybook-static --quiet"
+    "storybook:build": "storybook build -o out/storybook-static --quiet",
+    "fetch-entity": "NODE_OPTIONS='--disable-warning=ExperimentalWarning' node scripts/fetch-entity/src/index.ts"
```

Comment:
We can run `.ts` files in Node 23.6+ without the `--experimental-strip-types` flag, but it'll still warn you about it. 🤷 

Our `.nvmrc` says 18.17.0, but apparently I've been running on 23.11.0 accidentally for...a while? With no ill effect, so....... 🤷